### PR TITLE
FOUR-13186:Available processes Default option when user opens launchpad

### PR DESCRIPTION
**Pull request by fagubla**
## Issue & Reproduction Steps
As Participant/ Designer when open the Launchpad, it will show all the available processes available in the platform, and when user select a category it will show the processes of that category.
image

## How to Test
- Go to Processes
- Review the Process Browser

## Related Tickets & Packages
- [FOUR-13186](https://processmaker.atlassian.net/browse/FOUR-13186)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

[FOUR-13186]: https://processmaker.atlassian.net/browse/FOUR-13186?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-0aff120d84.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/CardProcess.vue [hunk 1]
```diff
@@ -95,6 +95,13 @@ export default {
      * Build URL for Process Cards
      */
     buildURL() {
+      if (this.category === undefined || this.category.id === -1) {
+        return "processes?status=ACTIVE"
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/CardProcess.vue:99]: Please use this API
`process_bookmarks/processes`,

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by pmPaulis** [comment]
@fagubla 
when a process was marked with favorities the **All Process** does not show the icon blue
image
image

File: resources/js/processes-catalogue/components/CardProcess.vue (hunk 1 above)
Comment (fagubla) [reply resources/js/processes-catalogue/components/CardProcess.vue:99]: @pmPaulis that API only return the bookmarked and not all processes

Comment (pmPaulis) [reply resources/js/processes-catalogue/components/CardProcess.vue:99]: This api works like the processes

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-0aff120d84.engk8s.processmaker.net

**Review by pmPaulis — APPROVED** [review]

**Comment by pmPaulis** [comment]
@fagubla now is working fine
image

**Pull request merged**